src/asHTML.ts

```diff
@@ -119,11 +125,11 @@ const wrapMapSerializerWithStringChildren = (
  * @returns HTML equivalent of the provided rich text or title field
  * @see Templating rich text and title fields from Prismic {@link https://prismic.io/docs/technologies/templating-rich-text-and-title-fields-javascript}
  */
-export const asHTML = (
-	richTextField: RichTextField | null | undefined,
+export const asHTML = <MaybeField extends RichTextField | null | undefined>(
+	richTextField: MaybeField,
```

Comment:
Following the above comment, `MaybeField` would just be `Field | null | undefined` if we move `null | undefined` to the parameter type. It would also clean up `AsHTMLReturnType`.

Reply:
Not possible, but renamed the `MaybeField`'s to just `Field`